db-scripts/10_initial_table_load.sql

```diff
@@ -222,3 +1228,98 @@ VALUES (1,'Route 1')
 ,(23,'Route 23')
 ,(24,'Route 24')
 ,(25,'Route 25');
+
+
+/*************************************\

```

Comment:
@tylerjamesf - this attempts to rebuild the games regions / routes / locations together. Not sure where gym / cities fit into the equation? Could those be merged into one and be typed? Or is it considered a "route"?